sailfishusermanagerinterface.h

```diff
@@ -16,7 +16,7 @@
 #define SAILFISH_USERMANAGER_DBUS_INTERFACE "org.sailfishos.usermanager"
 #define SAILFISH_USERMANAGER_DBUS_OBJECT_PATH "/"
 #define SAILFISH_USERMANAGER_GUEST_UID 105000
-#define SAILFISH_USERMANAGER_GUEST_HOME "/tmp/guest_home"
+#define SAILFISH_USERMANAGER_GUEST_HOME "/home/sailfish_guest"
```

Comment:
So both - and _ are removed from the username when creating home directory name for additional user. Hence, this cannot collide. Correct?

Reply:
Yes, only letters and numbers are allowed/used.